src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/integration/IntTestData.kt

```diff
@@ -1,8 +1,20 @@
 package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.integration
 
+import org.springframework.http.HttpHeaders
```

Comment:
This class, and AdjIntTestData, are a way of providing test data that is consistent between the API DB and Prison API.

It probably needs some refactoring, but IMO will make it far quicker to write int tests that involve the Prison API and the DB

Reply:
Coolio